src/exchanges.js

```diff
@@ -116,17 +123,19 @@ exchanges.declare({
 
 /** task group exchange */
 exchanges.declare({
-  exchange:           'task-group-defined',
-  name:               'taskGroupDefined',
-  title:              'GitHub release Event',
+  exchange:           'task-group-creation-requested',
+  name:               'taskGroupCreationRequested',
+  title:              'tc-gh requested the Queue service to create all the tasks in a group',
   description: [
-    'used for creating status indicators in GitHub UI using Statuses API',
+    'supposed to signal that taskCreate API has been called for every task in the task group',
+    'for this particular repo and this particular organization',
+    'currently used for creating initial status indicators in GitHub UI using Statuses API',
```

Comment:
If I'm understanding correctly, then, messages on this exchange have a primary routing key similar to the queue's exchanges, and a CC'd routing key containing either `route.checks` or `route.statuses`, and that's what the listeners use to distinguish which handler should handle a particular message.  Similar CC routes are used for messages from the queue's exchanges, by setting `task.routes`.

It might be worth including some information about that in the description here, to help the next person looking at the code :)